web-registry/src/hooks/useMsgClient.ts

```diff
@@ -73,7 +79,7 @@ export default function useMsgClient(
       if (_deliverTxResponse.code !== 0) {
         setError(_deliverTxResponse.rawLog);
         handleError();
-        return _deliverTxResponse.rawLog;
+        throw new Error(_deliverTxResponse.rawLog);
```

Comment:
I just realized that by throwing an error here we are going to execute `setError` and `handleError` twice which is not ideal. Two solutions here:
- we can surround `broadcast` call with a try-catch and return the error message in the catch to prevent that.
- remove both `setError` and `handleError` here since there are going to be executed in the upper catch

Reply:
Thanks for catching that, I've gone with the second option. Glad to adjust this part further if there is something better.